tests/unit_tests/storage/test_local_ensemble.py

```diff
@@ -66,7 +67,9 @@ def test_that_load_responses_throws_exception(tmp_path):
         experiment = storage.create_experiment()
         ensemble = storage.create_ensemble(experiment, name="foo", ensemble_size=1)
 
-        with pytest.raises(expected_exception=KeyError):
+        with pytest.raises(
+            expected_exception=ValueError, match="I_DONT_EXIST is not a response"
+        ):
```

Comment:
Doesn't it now check a different thing?

Reply:
Yes it does. The new check is more explicit I think.

Reply:
Though one cannot create a response, which is a response, but doesn't exist😅 Maybe this test is redundant? It is just a suggestion, feel free to disagree

Reply:
Not sure I follow. This makes sure `load_responses` provides a reasonable error message when users call it before responses are stored to storage, or using the wrong identifier.
Could you explain a bit more?

Reply:
If I understood correctly, we are trying to load response that does not exist. And the error message now says that what we are trying to load is not a response. Is that not the case?

Reply:
Yes, but given an instance of an `ensemble` object, we can't be sure up front whether the response we are requesting from it exists. I'm probably misunderstanding something here. Should we return a different message you think?

Reply:
I retract my comment, it is good to have this test, cause it does check a different behaviour than others. Would be nice for users sake to give an explicit message on when the response does not exist, but that is outside of scope of this PR